### PR TITLE
[TTPReq] Education: Accept contactless payments with only an iPhone

### LINKED ISSUE #14161: [TTPReq] Education: Accept contactless payments with only an iPhone
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

## Accept contactless payments with only an iPhone

- Use the visual assets from [Marketing Guidelines](https://developer.apple.com/tap-to-pay/marketing-guidelines/)
- Use the latest bezels
- Use localized images for all the supported TTP markets

## Designs

Image

### PR DESCRIPTION
**Pull request by staskus**
Closes: #14161

## Description

Configure the initial step of TTP education - 'Accept contactless payments with only an iPhone'.

I would like to confirm the factory approach first In the next PR, I will configure all the remaining steps (https://github.com/woocommerce/woocommerce-ios/pull/14605).

## Considerations

- Localization: Images have to be localized to the market
- Image size: I played with the compression of images since [the assets Apple provides are fairly large](https://developer.apple.com/tap-to-pay/marketing-guidelines/#editorial-guidelines).
- Country-specific configuration: should I consider injecting education configuration straight into `CardPresentPaymentsConfiguration` to make sure it's always added if we expand to different markets? For now, I decided to keep it separately. 

## Steps to reproduce

Prerequisites:
- Enable `tapToPayEducation` feature flag in `DefaultFeatureFlagService`
- TTP compatible iPhone
- Have US and UK located stores ready

### US

1. Select US store
2. Menu
3. Payments
4. About Tap to Pay
5. Confirm that the image shows currency based on the market ($ - US store)
6. Take a look at the image quality, let me know if you think it's good enough or should be better
7. (Ignore if the navigation bar is not visible in the view, it means you already have TTP set up. It won't be a problem with multiple steps visible) 

### UK

1. Select US store
2. Menu
3. Payments
4. About Tap to Pay
5. Confirm that the image shows currency based on the market (£ - UK store)
6. Confirm that UK limit information is shown and web view is opened
7. Take a look at the image quality, let me know if you think it's good enough or should be better
8. (Ignore if the navigation bar is not visible in the view, it means you already have TTP set up. It won't be a problem with multiple steps visible) 

## Testing information

Tested on iPhone 14 Pro 17.7.

## Screenshots

| US | UK | UK 2 |
| --- | --- | --- |
| IMG_9411 | IMG_9418 | IMG_9419 |

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on all devices (phone/tablet) and no regressions are added.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepsFactory.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        static let description = NSLocalizedString(
+            "tapToPay.education.step.intro.description",
+            value: "With Tap to Pay on iPhone and the Woo app, you can accept in-person, contactless payments, " +
+                   "right on your iPhone - from physical debit and credit cards, to Apple Pay and other digital " +
+                   "wallets - no extra hardware needed. It’s easy, secure, and private.",
+            comment: "Description for the initial Tap to Pay merchant education step. When using the name “Tap to Pay " +
+                     "on iPhone” in headlines or copy, do not shorten to “Tap to Pay” or “Apple Tap to Pay. Always " +
+                     "typeset “Tap to Pay on iPhone” as five words. The T and Ps should be uppercased, followed by " +
+                     "lowercase letters."
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepsFactory.swift:57]: Nice localization instruction 👍

**Review by jaclync — APPROVED** [review]
Looks great! :shipit: 

Just one question about this note about the 100 pound limit:

According to p1733312902624179/1733300788.131849-slack-C070SJRA8DP, this limit applies to card readers as well. Is it true that card readers have a higher limit? I tried checking the IPP docs but didn't find any mentions of the 100 pound limit for any reader type.

**Comment by staskus** [comment]
Thanks for the review!

> According to p1733312902624179/1733300788.131849-slack-C070SJRA8DP, this limit applies to card readers as well. Is it true that card readers have a higher limit? I tried checking the IPP docs but didn't find any mentions of the 100 pound limit for any reader type.

@jaclync,  from what I can see [in the documentation](https://support.stripe.com/questions/what-are-the-regional-contactless-limits-for-stripe-terminal-transactions) it applies to any contractless transaction:

> What are the regional contactless limits for Stripe Terminal transactions?
> United Kingdom: £100 GB

As I understand, card readers that allow inputting PINs have higher limits.

**Comment by jaclync** [comment]
> @jaclync, from what I can see [in the documentation](https://support.stripe.com/questions/what-are-the-regional-contactless-limits-for-stripe-terminal-transactions) it applies to any contractless transaction:
> 
> > What are the regional contactless limits for Stripe Terminal transactions?
> > United Kingdom: £100 GB
> 
> As I understand, card readers that allow inputting PINs have higher limits.

Ok I see, so it just means that transactions over 100 pounds require a pin code on WisePad 3 for example.

**Comment by joshheald** [comment]
@staskus @jaclync 

In the UK, generally speaking the limit is £100 if you tap a card. If you tap a device, there's (functionally) no limit.

Occasionally (actually I think only once), I've been asked for a PIN when I forget and tap a card for a transaction over £100. I asked round some friends and family a while ago and only the person who worked in a shop had seen that before, and for them it was almost never as well.

So in practice, most of the time if you tap a card over £100, the customer will just be asked to insert their card. I think that might differ now we have Tap to Pay on iPhone etc, but that's still quite rare to see in the wild.

**Pull request merged**